Braze Demo Match Game Content Extension/Info.plist

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unless it's an Apple convention I'm unaware of, I'd recommend avoiding spaces in the folder name